app/views/courses/EnrollmentsView.coffee

```diff
@@ -59,7 +62,12 @@ module.exports = class EnrollmentsView extends RootView
     @listenToOnce @classrooms, 'sync', @onceClassroomsSync
     @supermodel.trackRequest @classrooms.fetchMine()
     @prepaids = new Prepaids()
-    @supermodel.trackRequest @prepaids.fetchByCreator(me.id)
+    @supermodel.trackRequest @prepaids.fetchMineAndShared(me.id)
+    @listenTo @prepaids, 'sync', ->
+      @prepaids.each (prepaid) =>
+        prepaid.creator = new User()
+        if prepaid.get('creator') isnt me.id
+          @supermodel.trackRequest prepaid.creator.fetchCreatorOfPrepaid(prepaid)
```

Comment:
I think just putting a comment in here saying we don't include it because it's not being used is sufficient. That way if someone later wants to add it, they know it won't break anything to do that.